package-lock.json

```diff
@@ -7,6 +7,9 @@
 		"": {
 			"name": "pokeinfo",
 			"version": "0.0.1",
+			"dependencies": {
+				"svelte-icons-pack": "^2.1.0"
+			},
```

Comment:
add this to the "Sources" in the readme.md

- https://github.com/leshak/svelte-icons-pack
- https://leshak.github.io/svelte-icons-pack/#/